src/core/i18n/utils.js

```diff
@@ -230,6 +232,39 @@ function oneLineTranslationString(translationKey) {
   return translationKey;
 }
 
+type FormatFilesizeParams = {|
+  _filesize: typeof filesize,
+  i18n: I18nType,
+  size: number,
+|};
+
+// Translates a file size in bytes into a localized user-friendly format.
+export const formatFilesize = ({
+  _filesize = filesize,
+  i18n,
+  size,
+}: FormatFilesizeParams): string | null => {
+  const sizeStrings = {
+    // These are the expected values for the unit of measure returned by
+    // filesize. Realistically we shouldn't get anything back larger than TB.
+    B: i18n.gettext('%(formattedSize)s B'),
```

Comment:
I don't think this will be obvious to localizers so you should give each one of these an explantory comment. Example:
```js
const sizeStrings = {
  // translators: B is an abbreviation of Bytes in English. Localize it if necessary but use a short abbreviation.
  B: i18n.gettext('%(formattedSize)s B'),
  // translators: KB is an abbreviation of Kilobytes in English. Localize it if necessary but use a short abbreviation.
  KB: i18n.gettext('%(formattedSize)s KB'),
};
```